parking_permits/customer_permit.py

```diff
@@ -161,15 +167,17 @@ def create(self, address_id, registration):
                 is_user_of_vehicle = self.customer.is_user_of_vehicle(vehicle)
                 if not is_user_of_vehicle:
                     raise TraficomFetchVehicleError(
-                        f"Customer is not an owner or holder of a vehicle {registration}"
+                        _(
+                            f"Customer is not an owner or holder of a vehicle {registration}"
```

Comment:
Does this work? I dont think its a correct way to use translations with variables.
It should be ` _(
                    "Customer is not an owner or holder of a vehicle  %(registration)s"
                )
                % {
                    "registration": registration,
                }
            )`

Reply:
Fixed